spec/latest/triple-pattern-fragments/index.html

```diff
@@ -229,7 +229,7 @@
         <dt>hypermedia controls</dt>
         <dd>
           A Triple Pattern Fragment MUST contain hypermedia controls
-          that allow retrieving any other Triple Pattern Fragment of the same dataset.
+          that allow retrieval of any other Triple Pattern Fragment of the same dataset.
```

Comment:
Depends on writing style. I had learned from Simplified Technical English that use of gerunds is to be avoided as they can cause confusion for non-native readers (and can be awkward to translate). Downside is it leads to a 'drier' style, but if that aids comprehension then it's fine with me.